src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's contact in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidContact(String)}
+ */
+public class Contact {
+	
+	public static final String EXAMPLE = "123, some street";
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.